misc/testerina/modules/testerina-runtime/src/main/java/org/ballerinalang/test/runtime/util/TesterinaConstants.java

```diff
@@ -57,7 +57,7 @@ public class TesterinaConstants {
     public static final String TESTABLE = "testable";
     public static final String MODIFIED = "mod";
     public static final String CACHE_DIR = "cache";
-    public static final String JAVA_17_DIR = "java17";
+    public static final String JAVA_21_DIR = "java21";
```

Comment:
Shall we use a generic name for the constant e.g. `JAVA_PLATFORM_DIR`? so that we don't have to rename the variable for every update?